apps/docs/pages/guides/auth/sso/auth-sso-saml.mdx

```diff
@@ -225,7 +218,14 @@ For example, the following JSON structure configures attribute mapping for the `
 }
 ```
 
-You can include this structure in the `POST /auth/v1/admin/sso/providers` call under the `attribute_mapping` property.
+When creating or updating an identity provider with the [Supabase CLI](/docs/guides/cli) you can include this JSON as a file with the `--attribute-mapping /path/to/attribute/mapping.json` flag.
+
+For example, to change the attribute mappings to an existing provider you can use:
+
+```bash
+supabase sso update <provider-uuid> --project-ref <your-project> \
```

Comment:
On v1.46.1 the flag is labelled as  `--attribute-mapping-file` instead of `--attribute-mapping` do double check before merging

Reply:
![CleanShot 2023-03-30 at 20 12 18@2x](https://user-images.githubusercontent.com/8011761/228832094-9aa7f57b-c8d7-45e0-9efa-d53aacabc755.png) for quick reference

Reply:
should be `--attribute-mapping-file` based on the latest version